content/docs/how-we-work/security.md

```diff
@@ -1,37 +1,33 @@
 ---
 
 # Security
 
-Security is an everlasting concern in the world of software engineering. Understanding what security means in the context of our work and the various vectors that might be responsible for compromising our system's security is extremely important.
+Security is an everlasting concern in software engineering. Understanding security in the context of our work and recognizing various vectors that might compromise our system’s security is crucial. Security is a broad and deep topic, and it’s a team’s responsibility to implement safeguards at various levels.
 
-That said, security is a wide and deep topic, and a single person cannot hope to span the width and depth of this challenging area. It is the team's responsibility to understand security at various levels and implement safeguards.
+## Credentials Management
 
-For example, an application relies on database credentials to connect to the database. There can be API keys with which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them. Use environment variables or a vault for storing and retrieving them.
+Applications rely on database credentials and API keys to communicate with third-party services. These must not be kept in the repository or committed in Git history as they can be misused if exposed. Use environment variables or a vault for storing and retrieving them.
 
 ## Server and Website Security
 
-We are mindful of server and site security in our operations. This involves the use of sanitized databases, which entails the modification or removal of sensitive information to safeguard data privacy and security. Additionally, we routinely scrub unneeded data from our development environments, ensuring that any data no longer necessary for the project is removed and that the remaining data is securely stored.
+We ensure server and site security through several practices. Sanitized databases are used to protect data privacy and security by removing sensitive information. Additionally, we routinely scrub unneeded data from development environments, ensuring only necessary data is securely stored.
 
-To further enhance security, we establish separate accounts for each user on production and implement two-factor authentication wherever feasible. While we do not create test accounts on production, we are mindful to remove any such accounts promptly after testing is completed, if they were necessary in the first place.
+To enhance security, we establish separate accounts for each user on production and implement two-factor authentication wherever feasible. Test accounts on production are avoided or removed promptly after testing.
```

Comment:
It is important to say that we should not create test accounts on production unless required.